src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        String[] resultArray;
+        ArrayList<String> result = new ArrayList<>();
```

Comment:
```suggestion
        List<String> result = new ArrayList<>();
```
It's considered to be a good practice to use abstract type (interface)  as a reference

Reply:
done